src/backend/distributed/commands/statistics.c

```diff
@@ -209,6 +209,44 @@ PreprocessDropStatisticsStmt(Node *node, const char *queryString,
 }
 
 
+/*
+ * DropStatisticsObjectAddress returns the object address for drop statistic statement.
+ */
+ObjectAddress
+DropStatisticsObjectAddress(Node *node, bool missing_ok)
+{
+	DropStmt *dropStatisticsStmt = castNode(DropStmt, node);
+	Assert(dropStatisticsStmt->removeType == OBJECT_STATISTIC_EXT);
+
+	ObjectAddress objectAddress = { 0 };
+	List *objectNameList = NULL;
+	foreach_ptr(objectNameList, dropStatisticsStmt->objects)
+	{
+		RangeVar *stat = makeRangeVarFromNameList(objectNameList);
+
+		if (stat->schemaname == NULL)
+		{
+			Oid statsOid = get_statistics_object_oid(objectNameList,
```

Comment:
again, `get_statistics_object_oid` seems to be able to handle objects with/without schema names in `objectNameList`.

Can't we rely on that alone (e.g., do we really need `makeRangeVarFromNameList` ?)